homework2/PrefixTrie.cs

```diff
@@ -0,0 +1,155 @@
+﻿namespace PrefixTrie;
+
+/// <summary>
+/// this prefixTree.
+/// </summary>
+public class Trie
+{
+      private readonly TrieNode root = new TrieNode(' ');
```

Comment:
Не надо указывать тип дважды:
```suggestion
      private readonly TrieNode root = new(' ');
```
или
```suggestion
      private readonly var root = new TrieNode(' ');
```